dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7783,10 +7783,14 @@ public String toString() {
               + "reads and block writes) to wait for a response from the data server.")
           .build();
 
-  public static final PropertyKey DORA_CLIENT_READ_LOCATION_POLICY_ENABLED =
-      booleanBuilder(Name.DORA_CLIENT_READ_LOCATION_POLICY_ENABLED)
+  public static final PropertyKey DORA_ENABLED =
+      booleanBuilder(Name.DORA_ENABLED)
```

Comment:
Do you want to take one step further, to remove this config key and all condition checks based on this?

Reply:
Yes, I believe that's the plan. When we remove all code that runs under the assumption that Dora is **not** enabled, we can remove this config item.

Reply:
I do plan to, but not in this PR because the effort is non-trivial